.github/workflows/cypress-test-tenancy-disabled.yml

```diff
@@ -10,6 +10,7 @@ env:
   OPENSEARCH_SNAPSHOT_CMD: 'node ../scripts/opensearch snapshot'
   SPEC: 'cypress/integration/plugins/security-dashboards-plugin/aggregation_view.js,'
   PLUGIN_NAME: opensearch-security
+  OPENSEARCH_INITIAL_ADMIN_PASSWORD: myStrongPassword123!
 
 jobs:
   cypress-tests-multitenancy-disabled:
```

Comment:
not very important, but it would be nice if we get rid of the hard coded `sleep 500` of L75 :)

Reply:
Unfortunately I couldn't get this generalized out, since some cases don't want to start the dashboard (such as unit tests). Going to leave it as is for now since CI is blocked until this goes in, maybe we can introduce a health check of something so we are not always waiting 500.